notes/templates/core/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="/static/home_style.css">
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans|Oswald:300,400,700'></link>
+
+    </head>
```

Comment:
Lines 1-7 should go on `base.html` and in the body you can do 

{% block content %}{% endblock %}